js/scripts.js

```diff
@@ -2,16 +2,43 @@ function Phone(brand, price, color) {
 	this.brand = brand;
 	this.price = price;
 	this.color = color;
-}
+		this.getWarrantyCost = function() {
+		return this.price * 0.1;
+		console.log(this.getWarrantyCost);  //jak dodatkowo wyświetlić w konsoli wynik getWarrantyCost?
+		}; //sposób 1
```

Comment:
ten sposób jest słaby bo każdy jeden obiekt ma tworzoną własną funkcję, więc 100.000 obiektów niepotrzebnie stworzy nam 100.000 identycznych funkcji. Między innymi po to mamy prototypy, aby przedmioty stworzone w ramach jednego prototypu (lub wspólnego łańcucha prototypów) mogły implementować te same funkcje u siebie.

Co do pytania to najpierw log, potem `return` i będzie git